src/components/tfoot/tfoot-styles.jsx

```diff
@@ -1,14 +1,15 @@
 import styleUtils from '../table/style-utilities';
+import color from '../../styles/color';
```

Comment:
Don't directly import colors, it will break the theme. There should be a `color` field in the theme that you could use (although it might be better to use one of the variants or text/background colors instead?)

Reply:
Wasn't me :)  Jacob sat by my computer and told me how to fix it properly.